index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap" rel="stylesheet">
+    <title>Capstone - Sports League</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="media.css">
+    <script src="menu.js" defer></script>
+    <script src="speakers.js" defer></script>
+</head>
+
+<body>
+    <header>
+        <nav aria-label="Menu" class="top-header">
+            <ul>
+                <li><a class="f-icon" href="#"></a></li>
+                <li><a class="t-icon" href="#"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </nav>
+        <nav aria-label="Menu" class="mobile-menu">
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+            </ul>
+
+        </nav>
+        <nav aria-label="Menu" class="desktop-menu">
+            <a class="logo" href="index.html"></a>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#main">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><button type="button">Campaign</button></li>
+            </ul>
+        </nav>
+    </header>
+    <section id="banner" class="banner">
```

Comment:
- [ ] I sincerely commend your effort in personalizing your website, but I want to bring to your notice that you are expected to personalize the content of the website design guideline provided and not the design itself because I noticed you used a different background entirely for the hero section. I will encourage you to stick to the design guideline provided for you as much as possible 😄 


your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/233506549-b8e4b0bd-2f92-43bf-a969-62508418dddf.png)

design guidelines 👇🏽
![image](https://user-images.githubusercontent.com/63473584/233506651-3458b5d2-5d59-4f4b-9ac5-e5b971a9bb28.png)